sdk/textanalytics/cognitiveservices-textanalytics/review/cognitiveservices-textanalytics.api.md

```diff
@@ -0,0 +1,313 @@
+export type ExtractLinkedEntitiesResult = ExtractLinkedEntitiesSuccessResult | ExtractLinkedEntitiesErrorResult;
+
+// @public (undocumented)
+export interface ExtractLinkedEntitiesResultCollection extends Array<ExtractLinkedEntitiesResult> {
+    modelVersion: string;
+    statistics?: TextDocumentBatchStatistics;
+}
+
+// @public (undocumented)
+export interface ExtractLinkedEntitiesSuccessResult extends TextAnalyticsSuccessResult {
+    readonly entities: LinkedEntity[];
+}
+
+// @public
+export interface InnerError {
+    code: InnerErrorCode;
+    innerError?: InnerError;
+    message: string;
+    target?: string;
+}
+
+// @public
+export type InnerErrorCode = 'invalidParameterValue' | 'invalidRequestBodyFormat' | 'emptyRequest' | 'missingInputRecords' | 'invalidDocument' | 'modelVersionIncorrect' | 'invalidDocumentBatch' | 'unsupportedLanguageCode' | 'invalidCountryHint';
+
+// @public
+export interface LanguageInput {
+    // (undocumented)
+    countryHint?: string;
+    id: string;
+    // (undocumented)
+    text: string;
+}
+
+// @public
+export interface LinkedEntity {
+    dataSource: string;
+    id?: string;
+    language: string;
+    matches: Match[];
+    name: string;
+    url: string;
+}
+
+// @public
+export interface Match {
+    length: number;
+    offset: number;
+    score: number;
+    text: string;
+}
+
+// @public
+export interface MultiLanguageInput {
+    id: string;
+    language?: string;
+    text: string;
+}
+
+// @public (undocumented)
+export interface RecognizeEntitiesErrorResult extends TextAnalyticsErrorResult {
+}
+
+// @public (undocumented)
+export interface RecognizeEntitiesOptions extends TextAnalyticsOperationOptions {
+}
+
+// @public (undocumented)
+export type RecognizeEntitiesResult = RecognizeEntitiesSuccessResult | RecognizeEntitiesErrorResult;
+
+// @public (undocumented)
+export interface RecognizeEntitiesResultCollection extends Array<RecognizeEntitiesResult> {
+    modelVersion: string;
+    statistics?: TextDocumentBatchStatistics;
+}
+
+// @public (undocumented)
+export interface RecognizeEntitiesSuccessResult extends TextAnalyticsSuccessResult {
+    readonly entities: Entity[];
+}
+
+// @public (undocumented)
+export interface RecognizePiiEntitiesOptions extends TextAnalyticsOperationOptions {
+}
+
+// @public
+export interface SentenceSentiment {
+    length: number;
+    offset: number;
+    sentenceScores: SentimentConfidenceScorePerLabel;
+    sentiment: SentenceSentimentValue;
+    warnings?: string[];
+}
+
+// @public
+export type SentenceSentimentValue = 'positive' | 'neutral' | 'negative';
+
+// @public
+export interface SentimentConfidenceScorePerLabel {
+    // (undocumented)
+    negative: number;
+    // (undocumented)
+    neutral: number;
+    // (undocumented)
+    positive: number;
+}
+
+// @public
+export class TextAnalyticsClient {
+    constructor(endpointUrl: string, credential: TokenCredential | CognitiveServicesCredentials, options?: TextAnalyticsClientOptions);
```

Comment:
@bterlson How do you feel about `credential` vs `credentials`? All of our types from `@azure/identity` are singular, so I was preferring that.

Reply:
Brian confirmed singular is better. I've renamed `CognitiveServicesCredentials` to `CognitiveServicesCredential` to accomodate.

@suhas92 The reference docs are generated from the track 1 lib, they will get updated when we regenerate from this package. I don't have a strong pref as to `endpoint` vs `endpointUrl` though. Do you think it will not always be a URL?